scripts/bootstrap.py

```diff
@@ -0,0 +1,90 @@
+# Check python version
+import sys
+if not (sys.version_info.major == 3 and sys.version_info.minor >= 6):
+    print("This script requires Python 3.6 or higher!")
+    print("You are using Python {}.{}.".format(sys.version_info.major, sys.version_info.minor))
+    sys.exit(1)
+
+################################################################
+# Script flags
+import argparse
+parser = argparse.ArgumentParser()
+
+# Options:
+parser.add_argument("--debug", dest="is_debug", default=False, help="Enable debug build")
+
+# Parse arguments
+args = parser.parse_args()
+
+################################################################
+# Paths
+import os
+from os.path import abspath
+
+# Set cwd
+print("Changing current working directory to the repository root")
+repository_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+os.chdir(repository_dir)
+
+softconsole_path = "./iris-fsw-softconsole"
+libraries_path = "./iris-fsw-softconsole/Libraries"
+
+################################################################
+# Environment variables
+import find_os
+import os.path
+
+if find_os.is_windows:
+    import glob
+    microchip_toolchain_glob = glob.glob("C:/Microsemi/*/arm-none-eabi-gcc/bin")
```

Comment:
It is Microchip on my system. I think we should try both. 
```suggestion
    microchip_toolchain_glob = glob.glob("C:/Microchip/*/arm-none-eabi-gcc/bin") +  glob.glob("C:/Microsemi/*/arm-none-eabi-gcc/bin")
```